src/Microsoft.Azure.WebJobs.Extensions.Dapr/Services/DaprServiceClient.cs

```diff
@@ -53,6 +53,12 @@ private string GetDefaultDaprAddress(INameResolver resolver)
                 this.logger.LogDebug("DAPR_HTTP_PORT environment variable not found. Using port {daprPort} as default.", daprPort);
             }
 
+            var daprHost = resolver.Resolve("DAPR_HTTP_HOST");
```

Comment:
Please add use cases where DAPR_HTTP_HOST is applicable. Also, we need to add unit test for this condition.